### PR TITLE
[BH-1855] Custom sounds library for alarms

### PR DESCRIPTION
**Pull request by rrandomsky**
Implementation of custom alarm sounds feature, enabling users to upload their own audio files as alarm sounds via Mudita Center.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: image/system_a/data/lang/Deutsch.json [hunk 1]
```diff
@@ -52,9 +52,9 @@
     "app_bell_onboarding_shortcuts_step_restart": "F\u00fcr Ger\u00e4teneustart beide Seitenkn\u00f6pfe 10 Sek. dr\u00fccken",
     "app_bell_onboarding_shortcuts_step_rotate": "Zum Ausw\u00e4hlen drehen",
     "app_bell_onboarding_shortcuts_step_turn_off": "Zum Ausstellen des Ger\u00e4ts R\u00fcckseite 10 Sek. dr\u00fccken",
-    "app_bell_relaxation_error_message": "Nicht unterst\u00fctztes Dateiformat",
-    "app_bell_relaxation_file_deleted_message": "<text>Die Datei wurde gel\u00f6scht.</text>",
-    "app_bell_relaxation_limit_error_message": "<text>Datenlimit \u00fcberschritten.<br />Es k\u00f6nnte zu Fehlern kommen.</text>",
+    "app_bell_audio_error_message": "Nicht unterst\u00fctztes Dateiformat",
+    "app_bell_audio_file_deleted_message": "<text>Die Datei wurde gel\u00f6scht.</text>",
```
Comment (Lefucjusz) [inline image/system_a/data/lang/Deutsch.json:56]: Is this `<text> </text>` section needed here?

File: image/system_a/data/lang/Deutsch.json [hunk 2]
```diff
@@ -77,24 +77,26 @@
...
-    "app_bell_settings_alarm_settings_tone": "Alarmton",
+    "app_bell_settings_alarm_settings_chime": "Hauptalarmton",
+    "app_bell_settings_alarm_settings_sounds": "T\u00f6ne",
+    "app_bell_settings_alarm_settings_custom_sounds": "Eigene Kl\u00e4nge",
     "app_bell_settings_alarm_settings_volume": "Alarmlautst\u00e4rke",
-    "app_bell_settings_bedtime_settings_tone": "Klang",
-    "app_bell_settings_bedtime_settings_volume": "<text>Lautst\u00e4rke</text>",
-    "app_bell_settings_bedtime_tone": "Schlafens-Klang",
+    "app_bell_settings_bedtime_settings_tone": "Schlafenszeitton",
+    "app_bell_settings_bedtime_settings_volume": "<text>Schlafenszeit Lautst\u00e4rke</text>",
```
Comment (Lefucjusz) [inline image/system_a/data/lang/Deutsch.json:98]: The same as above

File: image/system_a/data/lang/English.json [hunk 3]
```diff
@@ -85,9 +85,9 @@
     "app_bell_onboarding_shortcuts_step_restart": "Press both side buttons for 10s to restart the device",
     "app_bell_onboarding_shortcuts_step_rotate": "Rotate to select",
     "app_bell_onboarding_shortcuts_step_turn_off": "Press back for 10s to turn off the device",
-    "app_bell_relaxation_error_message": "Unsupported media type",
-    "app_bell_relaxation_file_deleted_message": "<text>The file has been deleted.</text>",
-    "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may be displayed<br />correctly</text>",
+    "app_bell_audio_error_message": "Unsupported media type",
+    "app_bell_audio_file_deleted_message": "<text>The file has been deleted.</text>",
```
Comment (Lefucjusz) [inline image/system_a/data/lang/English.json:89]: The same as above

File: image/system_a/data/lang/Espanol.json [hunk 4]
```diff
@@ -51,9 +51,9 @@
     "app_bell_onboarding_shortcuts_step_restart": "Pulsa ambos botones laterales 10 s para reiniciar",
     "app_bell_onboarding_shortcuts_step_rotate": "Girar para seleccionar",
     "app_bell_onboarding_shortcuts_step_turn_off": "Pulsa Volver atr\u00e1s 10 s para apagar el dispositivo",
-    "app_bell_relaxation_error_message": "Formato de archivo no admitido",
-    "app_bell_relaxation_file_deleted_message": "<text>Se ha eliminado el archivo.</text>",
-    "app_bell_relaxation_limit_error_message": "<text>L\u00edmite de archivos alcanzado,<br />pueden producirse errores.</text>",
+    "app_bell_audio_error_message": "Formato de archivo no admitido",
+    "app_bell_audio_file_deleted_message": "<text>Se ha eliminado el archivo.</text>",
```
Comment (Lefucjusz) [inline image/system_a/data/lang/Espanol.json:55]: The same as above

File: image/system_a/data/lang/Francais.json [hunk 5]
```diff
@@ -53,9 +53,9 @@
     "app_bell_onboarding_shortcuts_step_restart": "Pressez les boutons lat\u00e9raux pendant 10 sec pour red\u00e9marrer",
     "app_bell_onboarding_shortcuts_step_rotate": "Faites tourner pour s\u00e9lectionner",
     "app_bell_onboarding_shortcuts_step_turn_off": "Appuyez sur Retour pendant 10 sec pour \u00e9teindre l'appareil",
-    "app_bell_relaxation_error_message": "<text>Format de fichier non pris en<br></br>charge</text>",
-    "app_bell_relaxation_file_deleted_message": "<text>Le fichier a \u00e9t\u00e9 supprim\u00e9.</text>",
-    "app_bell_relaxation_limit_error_message": "<text>Limite de fichiers exc\u00e9d\u00e9e.<br />Risque de probl\u00e8mes<br />d'affichage</text>",
+    "app_bell_audio_error_message": "<text>Format de fichier non pris en<br></br>charge</text>",
+    "app_bell_audio_file_deleted_message": "<text>Le fichier a \u00e9t\u00e9 supprim\u00e9.</text>",
```
Comment (Lefucjusz) [inline image/system_a/data/lang/Francais.json:57]: The same as above

File: image/system_a/data/lang/Polski.json [hunk 6]
```diff
@@ -52,9 +52,9 @@
     "app_bell_onboarding_shortcuts_step_restart": "Przytrzymaj oba boczne przyciski przez 10s, aby zrestartowa\u0107 urz\u0105dzenie",
     "app_bell_onboarding_shortcuts_step_rotate": "Obr\u00f3\u0107, aby wybra\u0107",
     "app_bell_onboarding_shortcuts_step_turn_off": "Przytrzymaj przycisk wstecz przez 10s aby wy\u0142\u0105czy\u0107 urz\u0105dzenie",
-    "app_bell_relaxation_error_message": "Nieobs\u0142ugiwany format pliku",
-    "app_bell_relaxation_file_deleted_message": "<text>Plik zosta\u0142 usuni\u0119ty.</text>",
-    "app_bell_relaxation_limit_error_message": "<text>Przekroczono limit plik\u00f3w.<br />Nie wszystkie pliki mog\u0105 by\u0107<br />wy\u015bwietlone poprawnie</text>",
+    "app_bell_audio_error_message": "Nieobs\u0142ugiwany format pliku",
+    "app_bell_audio_file_deleted_message": "<text>Plik zosta\u0142 usuni\u0119ty.</text>",
```
Comment (Lefucjusz) [inline image/system_a/data/lang/Polski.json:56]: The same as above

**Review by Lefucjusz — APPROVED** [review]

**Review by Maciej-Mudita — APPROVED** [review]

**Pull request merged**